index/server/pkg/server/endpoint.go

```diff
@@ -551,6 +551,7 @@ func buildIndexAPIResponse(c *gin.Context, indexType string, wantV1Index bool, p
 		maxSchemaVersion := params.MaxSchemaVersion
 		minVersion := params.MinVersion
 		maxVersion := params.MaxVersion
+		lastModified := params.LastModified
```

Comment:
Here I'd expect to be able to filter for min-max similar to the `minVersion`, `maxVersion`

Reply:
`minLastModified` and `maxLastModified` are now available. You can access them with `/v2index/all?minLastModified=<date>&maxLastModified=<date>` or any variation of `/v2index` with any combination of `minLastModified` or `maxLastModified`

Reply:
Nice! Let me test!

Reply:
I can confirm the `minLastModified` & `maxLastModified` are working :)